string type
int32 command_lenth
bool  break_flag
bool  wrong_flag
string data

